core/src/main/resources/jenkins/model/Jenkins/load-statistics.jelly

```diff
@@ -24,8 +24,7 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form" xmlns:i="jelly:fmt">
-  <l:layout title="${it.displayName} ${%Load Statistics}">
-    <st:include page="sidepanel.jelly" />
+  <l:layout title="${it.displayName} ${%Load Statistics}" type="one-column">
```

Comment:
Sidepanel with widgets looks like it would be useful specifically on this page, which you're probably visiting to diagnose queue/executor-related bottlenecks?

Reply:
Happy to restore this